air-script/src/test_utils/plonky3_traits.rs

```diff
@@ -116,35 +77,17 @@ where
         assert_eq!(x.into(), F::ZERO, "constraints had nonzero value on row {}", self.row_index);
     }
 
-    fn assert_eq<I1: Into<Self::Expr>, I2: Into<Self::Expr>>(&mut self, x: I1, y: I2) {
-        let x = x.into();
-        let y = y.into();
-        assert_eq!(x, y, "values didn't match on row {}: {} != {}", self.row_index, x, y);
-    }
-}
-
-impl<'a, F, EF> AirBuilderWithPublicValues for DebugConstraintBuilderWithAirScriptTraits<'a, F, EF>
-where
-    F: Field,
-    EF: ExtensionField<F>,
-{
-    type PublicVar = Self::F;
-
-    fn public_values(&self) -> &[Self::F] {
+    fn public_values(&self) -> &[Self::PublicVar] {
         self.public_values
     }
-}
-
-impl<'a, F, EF> ExtensionBuilder for DebugConstraintBuilderWithAirScriptTraits<'a, F, EF>
-where
-    F: Field,
-    EF: ExtensionField<F>,
-{
-    type EF = EF;
 
-    type ExprEF = EF;
+    fn periodic_evals(&self) -> &[<Self as MidenAirBuilder>::PeriodicVal] {
+        self.periodic_columns.as_slice()
```

Comment:
Should this depend on the current row? I guess for debugging this is fine, but the values returned will depend on the row modulo the period.